l5kit/l5kit/data/zarr_dataset.py

```diff
@@ -70,23 +70,26 @@ def __init__(self, path: str, key: str = ""):
         if ".zarr" not in self.path:
             print("zarr dataset path should end with .zarr (for now). Open will fail for this dataset!")
 
-    def initialize(self, mode: str = "w") -> None:
+    def initialize(self, mode: str = "w", scenes_len: int = 0, frames_len: int = 0, agents_len: int = 0) -> None:
```

Comment:
Minor, maybe scene_number?
scene length sounds like how many frames are in a scene